src/Microsoft.DotNet.ProjectModel/project.json

```diff
@@ -2,15 +2,14 @@
     "version": "1.0.0-*",
     "description": "Types to model a .NET Project",
     "dependencies": {
-        "Microsoft.CSharp": "4.0.1-beta-23504",
-        "System.Collections": "4.0.11-beta-23504",
-        "System.Linq": "4.0.1-beta-23504",
-        "System.Threading.Thread": "4.0.0-beta-23504",
-        "System.Runtime.Loader": "4.0.0-beta-23504",
-        "System.Dynamic.Runtime": "4.0.11-beta-23504",
-        "System.Runtime.InteropServices.RuntimeInformation": "4.0.0-beta-23504",
-        "System.Security.Cryptography.Algorithms": "4.0.0-beta-23504",
-        "System.Xml.XDocument": "4.0.11-beta-23504",
+        "NETStandard.Library": "1.0.0-rc2-23608",
+        "System.Linq": "4.0.1-rc2-23608",
+        "System.Threading.Thread": "4.0.0-rc2-23608",
+        "System.Runtime.Loader": "4.0.0-rc2-23608",
+        "System.Dynamic.Runtime": "4.0.11-rc2-23608",
+        "System.Security.Cryptography.Algorithms": "4.0.0-rc2-23608",
+        "Microsoft.CSharp": "4.0.1-rc2-23608",
```

Comment:
Ok, probably point-in-time. All of that discussions are separate from the PR itself though (just didn't have anywhere else to complain :stuck_out_tongue_closed_eyes:)